bin/mark_changepoints_in_json

```diff
@@ -173,24 +173,42 @@ class Segments(object):
             if math.fabs(self.segments[0].mean - self.segments[1].mean) < self.delta:
                 self._merge_segment(0)
 
+    def _is_segment_a_slowdown(self, index, pc_bound=0.1):
+        """Test whether mean of segment index is slower than the last segment
+        mean by >= pc_bound percent.
+        """
+        if self.segments[index].mean <= self.segments[-1].mean:
+            return False  # Segment index is faster than segment -1.
+        pc = math.fabs(self.segments[index].mean - self.segments[-1].mean /
+                       self.segments[-1].mean * 100.0)
+        return pc >= pc_bound
+
     def get_classification(self):
         """Return the classification for this run sequence.
         """
+        PC_BOUND = 0.01  # TODO: Factor out as a CLI option before merging with master.
         classification = ''
         # Flat -> no changepoints.
         if len(self.segments) == 1:
             classification = 'flat'
+        # Warmup (a) -> only two segments and second is faster.
+        elif len(self.segments) == 2 and self.segments[1].mean < self.segments[0].mean:
```

Comment:
We probably don't *need* these two clauses, as such, but the intention was to deal with a couple of common cases (obviously the warmup case is much more common than the 2-segment slowdown) and avoid calculating the list on line 205 when we don't need it.